src/lib.rs

```diff
@@ -11,8 +14,8 @@ mod scheduler_longterm;
 pub use scheduler_longterm::LongtermScheduler;
 
 mod models;
-pub use models::{Card, Rating, RecordLog, ReviewLog, SchedulingInfo, State};
+pub use models::{Card, Rating, ReviewLog, SchedulingInfo, State};
```

Comment:
`RecordLog` is needed here because binding.

for example

https://github.com/open-spaced-repetition/rs-fsrs-c/blob/9081374d011c2ad87f6d97320a45dbe0e5e15c8f/src/fsrs_def.rs#L114-L121

Reply:
fixed
untrack the changes in #32